Makefile

```diff
@@ -15,7 +15,7 @@ docker-generate-examples:
 	$(DOCKER_RUN_ARGS) bash -c "python3 setup.py develop && python3 tools/generate_code_examples.py generated_code_examples"
 
 docker-flake8:
-	$(DOCKER_RUN_ARGS) bash -c "flake8 ."
+	$(DOCKER_RUN_ARGS) bash -c "flake8 --exclude tmp,generated_code_examples ."
```

Comment:
Maybe sync with 
https://github.com/BayesWitnesses/m2cgen/blob/e0bae16a64c2ac28abd8fe64f1c926c2f8a539db/setup.cfg#L6-L7
?

Reply:
Hm, weird. Not sure how I overlooked that and why the config wasn't picked up in my local env. Thanks for pointing this out